lib/contacts_service.dart

```diff
@@ -222,13 +223,25 @@ class Contact {
       this.givenName,
       this.identifier,
       this.jobTitle,
-      this.accountType,
+      this.androidAccountType,
       this.middleName,
       this.prefix,
       this.suffix,
       this.birthday,
     ].where((s) => s != null));
   }
+
+  AndroidAccountType accountTypeFromString(String androidAccountType) {
+    if (androidAccountType == null) return null;
+    if (androidAccountType.contains("com.google")) {
+      return AndroidAccountType.google;
+    } else if (androidAccountType.contains("com.whatsapp")) {
+      return AndroidAccountType.whatsapp;
+    } else if (androidAccountType.contains("com.facebook")) {
+      return AndroidAccountType.facebook;
+    }
+    return AndroidAccountType.other;
```

Comment:
Make a comment that there are many other account types on Android (such as Samsung etc) and we do not currently support them.

Reply:
Done.